### ISSUE TITLE
GBIF participant page (i.e. country pages)

### ISSUE THREAD

**Issue by timrobertson100** [gbif/portal16#70]
To replace the current country pages, simplifying them into a single page.

Most likely with less repetition than the current design, but capturing all the key information.  Likely to contain a toggle to select data about (default) vs. data from.  

Likely to make use of cards and links to search results rather than repeating the same info in various wells.

**Comment by kbraak** [comment]
@kcopas 

I'd like to propose that we add a section to the Participant page aimed at publishers that explains how the Participant supports publishing and data hosting withing their country/network. 

**Background:** 

The [BID data hosting quick guide](http://bid.gbif.org/en/community/data-hosting/) encourages publishers to try and discover "whether the national GBIF node offers a data hosting solution." Sadly the guide will most likely leave publishers hanging at this step: the guide directs publishers to the [GBIF participants list](http://www.gbif.org/participation/participant-list) and after opening the GBIF Participant page they won't find any information about how that Participant offers a data hosting solution.

A list of Country Participants and how they support publishing and data hosting can currently be found on the [list of  Trusted IPT Hosting Centres](https://github.com/gbif/ipt/wiki/dataHostingCentres#data-hosting-centres-by-country). This information would make a valuable addition to the Participant page. If a Participant has no data hosting solution this should also be make clear. If a Participant such as France provides data hosting solutions to a number of other countries, this should be apparent not only from France's page but from each of the countries' pages that they support.

**Comment by kcopas** [comment]
In principle, yes—but this would most likely be a node function per se, rather than a participant one. I'd like to hear what @burkeker says about the prospect of adding another chunk of additional content to what I think is already a bit of a creaky content model. Propose no regression for EOY release, but additions only after careful consideration.

**Comment by MortenHofft** [comment]
I like the idea, but it doesn't sound like something to model in Drupal. It sounds more like a registry thing. sounds like the api need to be expanded to do this in a nice way. Currently all IPT stats are static. It would be nice to have them dynamic and would be nice to have them searchable by coverage. this installation covers these countries. these countries are covered by these installations belonging to these participants.
There might already be a way, but no good one I can come up with of the top of my head. What you say @kbraak , what would be the best way to do this in your opinion? You know the APIs and content better than I.

**Comment by kcopas** [comment]
Another +1 for the idea. Of course, I'm useless w.r.t. the implementation.

Is this content for the EOY release? Either way, an excellent new
requirement ;-)

=====
*Kyle Copas*
Niels W. Gades Gade 55
2100 Copenhagen Ø
(+45) 28 75 14 75 | skype kylecopas

**Comment by MortenHofft** [comment]
https://demo.gbif.org/country/AU/summary
Please create an new issue if there are important missing in the implementation
